### DISCUSSION TITLE
Draft PRs should not tag CODEOWNERS for review

### DISCUSSION THREAD

**Discussion by jamesmcguirepro** [community/community#58270] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When a draft PR is opened, the `CODEOWNERS` should not automatically tagged for code review.

Instead, the `CODEOWNERS` should be tagged for code review when the PR is ready for review.

When I open a Draft PR, it's still requires some finishing touches and will be noisy for anyone tagged for review.  

Instead, GitHub should wait until the PR is marked as "Ready for Review".  This will spare the `CODEOWNERS` teams excessive emails/review notifications.

**Comment by grumpyoldman-io** [comment]
Do you perhaps have the branch protection rule enabled for code owners? https://github.blog/news-insights/product-news/introducing-draft-pull-requests/ it says that this shouldnt happen:

> Also, if you have a [CODEOWNERS file](https://github.blog/2017-07-06-introducing-code-owners/) in your repository, a draft pull request will suppress notifications to those reviewers until it is marked as ready for review.

I'm running in to the same issue, wondering if its a bug or a confusing feature

**Comment by jamesmcguirepro** [comment]
We had a branch protection rule for `CODEOWNERS`, but have since disabled it.

**Comment by taterhead247** [comment]
It's not sending email notifications to codeowners. But it is showing that they've been requested. Which is kinda annoying.
